{
    "name": "pillow",
    "cve_id": "CVE-2021-28676",
    "publish_date": "2021-06-02T16:15Z",
    "cwe_id": "CWE-400",
    "owner": "Alex Clark",
    "affected_versions": [
        {
            "version_value": "8.2.0",
            "operator": "<"
        }
    ],
    "references": [
        "https://pillow.readthedocs.io/en/stable/releasenotes/8.2.0.html#cve-2021-28676-fix-fli-dos",
        "https://github.com/python-pillow/Pillow/pull/5377",
        "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MQHA5HAIBOYI3R6HDWCLAGFTIQP767FL/"
    ],
    "description": "An issue was discovered in Pillow before 8.2.0. For FLI data, FliDecode did not properly check that the block advance was non-zero, potentially leading to an infinite loop on load.",
    "latest_version": "8.2.0",
    "language": "python",
    "license": "HPND",
    "repository_summary": "An extensive library providing additional facitilities to the Python Standard Library.",
    "repo_url": "https://github.com/python-pillow/Pillow",
    "impact": {
        "cvss2_score": null,
        "cvss2_severity": null,
        "vector_string": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
        "av": "NETWORK",
        "ac": "LOW",
        "pr": "NONE",
        "c": "NONE",
        "i": "NONE",
        "a": "HIGH",
        "impact_score": 3.6,
        "exploitability_score": 3.9,
        "cvss3_score": 7.5,
        "cvss3_severity": "HIGH",
        "ui": "NONE",
        "s": "UNCHANGED"
    },
    "ochrona_severity_score": 7.5
}